### PR TITLE
Updates debug outfit and ports omnitool

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Ports: https://github.com/tgstation/tgstation/pull/46590 and https://github.com/tgstation/tgstation/pull/48486 
Also I modified admin RCD to have upgrades and be faster.

## Why It's Good For The Game

On the list + no reason to not have it 

## Changelog
:cl:  kriskog/Fikou ported by Archanial
add: Omnitool.
add: Debug outfit has been updated and adjusted.
balance: Admin RCD is superfast now and has upgrades.
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/boxes.dm [hunk 1]
```diff
@@ -1143,3 +1143,60 @@
 		/obj/item/stock_parts/matter_bin = 2,
 		/obj/item/screwdriver = 1)
 	generate_items_inside(items_inside,src)
+
+/obj/item/storage/box/material
```
Comment (PowerfulBacon) [inline code/game/objects/items/storage/boxes.dm:1147]: We already have this under closet/syndicate/materials

Comment (Archanial) [reply code/game/objects/items/storage/boxes.dm:1147]: You mean closet/syndicate/resources?
Closet is closet, box is a box, can't spawn closet in backpack too.

File: code/game/objects/items/debug_items.dm [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+/obj/item/debug/human_spawner
+	name = "human spawner"
+	desc = "Spawn a human by aiming at a turf and clicking. Use in hand to change type."
+	icon = 'icons/obj/guns/magic.dmi'
+	icon_state = "nothingwand"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	w_class = WEIGHT_CLASS_SMALL
+	var/datum/species/selected_species
+	var/valid_species = list()
```
Comment (PowerfulBacon) [inline code/game/objects/items/debug_items.dm]: Is this ever used.

Comment (Archanial) [reply code/game/objects/items/debug_items.dm]: No, don't think so

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**